### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ricdikulous**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,15 +18,27 @@ This project contains a number of components:
...
+## Running Locally
+To run the application locally you need 3 components running.
+* An Algorand LocalNet
+* The [Write-through cache Algorand IPFS gateway](./src/voting-metadata-api/README.md)
+* The [Voting dApp](./src/dapp/README.md)
 
+Here are the steps you can follow to run the application locally:
 1. Install `AlgoKit` - [Link](https://github.com/algorandfoundation/algokit-cli#install): Ensure you can execute `algokit --version`.
 2. Run `algokit localnet start` to start a LocalNet network
-3. Run `algokit bootstrap all` in `src` to install dependencies and set up `.env` files
```
Comment (robdmoore) [inline README.md:25]: The instructions on the left are correct and probably faster, plus mention the VS Code F5 option, should we keep  or augment them?

File: README.md [hunk 2]
```diff
@@ -52,7 +64,8 @@ To use it you need to configure the following with a `DEV_` prefix for TestNet a
     * `IPFS_GATEWAY_URL`: The URL to the IPFS gateway that is deployed as part of this solution including the `/ipfs` e.g. `https://api.testnet.voting.algorand.foundation/ipfs`
     * `ALGO_EXPLORER_URL`: The URL to AlgoExplorer e.g. `ALGO_EXPLORER_URL`
     * `NFT_EXPLORER_URL`: The URL to NFT Explorer for an asset, minus the asset ID e.g. `https://nftexplorer.app/asset/`
-    
+    * `IS_TESTNET`: `true` or `false`
+    * `CREATOR_ALLOW_LIST_ADDRESSES`: The allowlist for voting round creators. To keep it open for anyone set it to `any`. Alternatively you can limit access to an allowist of addresses e.g. `MOIL6NTBHUFAWV5TYY6YYRJ2N3LOAPOBEV4ZPFZAJKZX3OHGQVMYLEHEUU,ODTX32FQL44D5GIJ2CMCEZ4G3FGUU3WUYDHJZDRNSSLHDO54ESGKXC25UQ`
```
Comment (robdmoore) [inline README.md:68]: Is there also a web3 storage one now too?

File: README.md (hunk 1 above)
Comment (ricdikulous) [reply README.md:25]: From the existing, I get errors when I run step 3 and step 4 is unnecessary to run the application locally

File: README.md (hunk 2 above)
Comment (ricdikulous) [reply README.md:68]: If you mean `WEB3_STORAGE_API_TOKEN`, it's already there, listed above under `Secrets`

File: README.md (hunk 1 above)
Comment (robdmoore) [reply README.md:25]: It's not if running via VS Code, but I will change that.

**Pull request merged**